src/core/ext/filters/client_channel/client_channel.cc

```diff
@@ -142,14 +142,10 @@ class ChannelData {
     return disconnect_error_.Load(MemoryOrder::ACQUIRE);
   }
 
-  Mutex* data_plane_mu() const { return &data_plane_mu_; }
-
-  LoadBalancingPolicy::SubchannelPicker* picker() const {
-    return picker_.get();
-  }
-  void AddQueuedPick(QueuedPick* pick, grpc_polling_entity* pollent);
-  void RemoveQueuedPick(QueuedPick* to_remove, grpc_polling_entity* pollent);
-
+  Mutex* resolution_mu() const { return &resolution_mu_; }
+  void AddResolverQueuedCall(QueuedCall* call, grpc_polling_entity* pollent);
```

Comment:
Please clarify that these functions need to hold a lock (which I think is true based on my read of the code)

Reply:
I've added a comment about this.